src/css/_navigation.scss

```diff
@@ -209,7 +225,7 @@ input {
       width: 100%;
       padding: 0;
       margin:0;
-    
```

Comment:
I added a space here but it was showing as an additional space, removing it now shows red sign. What is it expecting? @kingthorin

Reply:
It's missing spaces.